src/Components/Home/Nav.js

```diff
@@ -1,9 +1,127 @@
-import React from 'react'
+import React from "react";
+import Logo from "../../Logo";
 
 export default function Nav() {
-    return (
-        <div>
-            Navbar
+  
+  return (
+    <div class="relative bg-white">
```

Comment:
@shrushti2000 Header background should be transparent. 

Now:
![image](https://user-images.githubusercontent.com/43892590/104897020-e57f6100-599d-11eb-99df-782cbe638e90.png)

My Suggestion:
![image](https://user-images.githubusercontent.com/43892590/104897119-01830280-599e-11eb-8863-c62d97a0af48.png)